src/app/shared/stache/modules/video/video.component.html

```diff
@@ -0,0 +1,3 @@
+<div class="stache-video">
+  <iframe class="stache-video-embed" [src]="src" frameborder="0" allowfullscreen></iframe>
```

Comment:
Last comment, then this can be merged: 
- Move the closing iframe tag to a new line
- Rename class to `stache-video-object` to be consistent with the image component

This looks good!